roles/irods_davrods/templates/config/davrods-vhost.conf

```diff
@@ -6,10 +6,11 @@
 # Below we provide an example vhost configuration that enables davrods using
 # its default options.
 #
+ServerName {{ hostvars[inventory_hostname]['fqdn'] }}
+
 <VirtualHost *:443>
 
     # Enter your server name here.
```

Comment:
Forgot to relocate comment for relocated line?

Reply:
I changed the comment also. Thanks.